lib/screens/mapPage/ExploreScreen.js

```diff
@@ -1,26 +1,26 @@
-import React, { useEffect, useState, useRef, useCallback } from "react";
+import Constants from "expo-constants";
+import * as Location from "expo-location";
+import React, { useCallback, useEffect, useRef, useState } from "react";
 import {
+  Animated,
+  Dimensions,
+  Image,
+  Keyboard,
+  Platform,
   StyleSheet,
   Text,
   TextInput,
-  View,
-  Animated,
-  Image,
   TouchableOpacity,
-  Dimensions,
-  Platform,
+  View,
 } from "react-native";
 import MapView, { Marker } from "react-native-maps";
-import * as Location from "expo-location";
-import NoteDetailModal from "./NoteDetailModal";
-import { formatToLocalDateString } from "../../components/time";
 import Ionicons from "react-native-vector-icons/Ionicons";
 import MaterialCommunityIcons from "react-native-vector-icons/MaterialCommunityIcons";
-import { mapDarkStyle, mapStandardStyle } from "./mapData";
-import ApiService from "../../utils/api_calls";
-import Constants from "expo-constants";
-import { Keyboard } from "react-native";
 import { useTheme } from "../../components/ThemeProvider";
+import { formatToLocalDateString } from "../../components/time";
+import ApiService from "../../utils/api_calls";
+import NoteDetailModal from "./NoteDetailModal";
+import { mapDarkStyle, mapStandardStyle } from "./mapData";
```

Comment:
These changes look like rearrangement of imports, is there a purpose behind this?

Reply:
@kate-holdener This issue was not created by me. I reached out to Nikhil, but I haven't received a response from him. Issue #162 was created by me.